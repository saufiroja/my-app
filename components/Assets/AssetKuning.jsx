import React from "react";
import Image from 'next/image';

const AssetKuning = () => {
  return (
    <div className="absolute xl:ml-[-215px] xl:mt-[-185px] xl:flex lg:ml-[-215px] lg:mt-[-185px] lg:flex md:ml-[-215px] md:mt-[-185px] md:flex sm:ml-[-215px] sm:mt-[-185px]">
      <Image src="/images/kuning.svg" alt="Asset-Kuning" width={280} height={280}/>
    </div>
  );
};

export default AssetKuning;