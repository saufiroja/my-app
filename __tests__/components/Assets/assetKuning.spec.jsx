import React from 'react'
import { render, screen } from '@testing-library/react'
import AssetKuning from '../../../components/Assets/AssetKuning'
import renderer from "react-test-renderer"

describe('AssetKuning', () => {
    it('renders a image', () => {
    render(<AssetKuning />)

    const image = screen.getByRole('img', {
        name: "Asset-Kuning",
        })

    const incrediblesPosterImg = screen.getByAltText('Asset-Kuning')

        expect(image).toBeInTheDocument()
        expect(incrediblesPosterImg).toBeInTheDocument()
    })

    it('matches the snapshot', () => {
    const tree = renderer.create(<AssetKuning />).toJSON()

    expect(tree).toMatchSnapshot()
    })
})