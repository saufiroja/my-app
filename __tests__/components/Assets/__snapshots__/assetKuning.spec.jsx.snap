// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssetKuning matches the snapshot 1`] = `
<div
  className="absolute xl:ml-[-215px] xl:mt-[-185px] xl:flex lg:ml-[-215px] lg:mt-[-185px] lg:flex md:ml-[-215px] md:mt-[-185px] md:flex sm:ml-[-215px] sm:mt-[-185px]"
>
  <span
    style={
      Object {
        "background": "none",
        "border": 0,
        "boxSizing": "border-box",
        "display": "inline-block",
        "height": "initial",
        "margin": 0,
        "maxWidth": "100%",
        "opacity": 1,
        "overflow": "hidden",
        "padding": 0,
        "position": "relative",
        "width": "initial",
      }
    }
  >
    <span
      style={
        Object {
          "background": "none",
          "border": 0,
          "boxSizing": "border-box",
          "display": "block",
          "height": "initial",
          "margin": 0,
          "maxWidth": "100%",
          "opacity": 1,
          "padding": 0,
          "width": "initial",
        }
      }
    >
      <img
        alt=""
        aria-hidden={true}
        src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27280%27%20height=%27280%27/%3e"
        style={
          Object {
            "background": "none",
            "border": 0,
            "display": "block",
            "height": "initial",
            "margin": 0,
            "maxWidth": "100%",
            "opacity": 1,
            "padding": 0,
            "width": "initial",
          }
        }
      />
    </span>
    <img
      alt="Asset-Kuning"
      data-nimg="intrinsic"
      decoding="async"
      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
      style={
        Object {
          "border": "none",
          "bottom": 0,
          "boxSizing": "border-box",
          "display": "block",
          "height": 0,
          "left": 0,
          "margin": "auto",
          "maxHeight": "100%",
          "maxWidth": "100%",
          "minHeight": "100%",
          "minWidth": "100%",
          "objectFit": undefined,
          "objectPosition": undefined,
          "padding": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": 0,
        }
      }
    />
    <noscript>
      <img
        alt="Asset-Kuning"
        data-nimg="intrinsic"
        decoding="async"
        loading="lazy"
        src="/_next/image?url=%2Fimages%2Fkuning.svg&w=640&q=75"
        srcSet="/_next/image?url=%2Fimages%2Fkuning.svg&w=384&q=75 1x, /_next/image?url=%2Fimages%2Fkuning.svg&w=640&q=75 2x"
        style={
          Object {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
    </noscript>
  </span>
</div>
`;
